### PR TITLE
fix: Post/Put->Create/Update and fix enum on `associations` body param

### PR DESCRIPTION
**Pull request by zkirby**

### CODE REVIEW

File: docs/pages/beta/crm/crm-beta.yaml [hunk 1]
```diff
@@ -179,10 +179,10 @@ paths:
                   type: array
                   items:
                     type: string
-                    enum: [accounts, deals, leads, notes, tasks, events, eventAttendees, calls, emails]
                   description: |
                     Array of associations to include in the response.
-                    OwnerUserId is always included.
+                    Allowed values: accounts, deals, leads, notes, tasks, events, eventAttendees, calls, emails. 
+                    ListOwnerUserId is always included in the response in the response.
```
Comment (averyyip) [inline docs/pages/beta/crm/crm-beta.yaml:185]:
```suggestion
                    OwnerUserId is always included in the response in the response.
```

Comment (averyyip) [reply docs/pages/beta/crm/crm-beta.yaml:185]: Don't forget to fix the rest

File: docs/pages/beta/crm/crm-beta.yaml [hunk 2]
```diff
@@ -179,10 +179,10 @@ paths:
                   type: array
                   items:
                     type: string
-                    enum: [accounts, deals, leads, notes, tasks, events, eventAttendees, calls, emails]
                   description: |
                     Array of associations to include in the response.
-                    OwnerUserId is always included.
+                    Allowed values: accounts, deals, leads, notes, tasks, events, eventAttendees, calls, emails.
```
Comment (averyyip) [inline docs/pages/beta/crm/crm-beta.yaml:184]: Just confirming here that we've verified the list of associations allowed for each object?

Also, are you planning to fix the examples in a separate pr or the same pr?

**Review by averyyip — COMMENTED** [review]
Looking good, we're getting close!

File: docs/pages/beta/crm/crm-beta.yaml (hunk 2 above)
Comment (zkirby) [reply docs/pages/beta/crm/crm-beta.yaml:184]: Yeah, will post a follow up in the slack thread - but technically these lists are correct as of now. 

Also which example?

Comment (averyyip) [reply docs/pages/beta/crm/crm-beta.yaml:184]: The example responses  for the associations field
https://docs.vessel.dev/pages/beta/crm/api-reference/get-all-contacts
image

**Review by averyyip — APPROVED** [review]

**Pull request merged**